src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -83,6 +85,10 @@ $header-height: 45px;
     &__second-icon {
       margin-left: 5px;
     }
+
+    @media (min-width: $large-laptop-width) {
+      display: none;
+    }
```

Comment:
What I see looks good AS AN OPTION, and a good OPTION is to lock it open - but is it possible to hide / minimise this by default? 

In practice, only some people want to have a chat function - and others feel it takes away from the "focus of the experience."  Most people are not on Sparkle to have detailed text based chats with others - but seeing other peoples comments in spaces has a part.

Reply:
imho when we are facing the problem to handle logic related to keeping the sidebar 'locked open' - we will update this .css 
But right now it's fine solution I suppose

Reply:
@yurii-lubynets The changes in this PR are basically a version of the 'locked open' stuff

@Sparkle-T While I hear your concerns, they aren't aligned with the issue described in https://github.com/sparkletown/internal-sparkle-issues/issues/622, which is what this PR is working on. Until @sparkletown/product says otherwise, we are continuing as planned.

If you feel strongly about this (which it seems you do), then please chase up https://github.com/sparkletown/internal-sparkle-issues/issues/622#issuecomment-814613821 and get a decision made.

Until then, @yurii-lubynets please disregard the @Sparkle-T 's comments above and continue as planned.

(@jonboutelle @mobilevinay FYI RE: 'too many cooks spoil the stew')